### PR TITLE
Lock puppet-strings to less than 2.0 if on Ruby 2.0

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,7 +3,11 @@ source 'https://rubygems.org'
 gem 'kafo', '>= 3.0.0'
 gem 'librarian-puppet'
 gem 'puppet', ENV.key?('PUPPET_VERSION') ? "~> #{ENV['PUPPET_VERSION']}" : '~> 6.0'
-gem 'puppet-strings'
+if RUBY_VERSION > '2.0'
```
Comment (mmoll) [inline Gemfile]: I think `if RUBY_VERSION < '2.1'` should be used here.

**Review by mmoll — CHANGES_REQUESTED** [review]

**Comment by mmoll** [comment]
`semverse` also needs a pinning for `< 2.1` and another one for `< 2.2` in my clean installs to work.

**Comment by ehelms** [comment]
@mmoll I can add semverse -- what versions of ruby correpond to each of those?

**Comment by mmoll** [comment]
`< 2` for Ruby `< 2.1` and `< 3` for Ruby `< 2.2`

**Comment by ehelms** [comment]
@mmoll Updated, check me I got that right

File: Gemfile [hunk 2]
```diff
@@ -14,5 +18,11 @@ end
 group :development do
   # Needed to pin dependencies
   gem 'puppet_forge'
-  gem 'semverse'
+  if RUBY_VERSION < '2.1'
+    gem 'semverse', '< 2'
+  elsif
```
Comment (mmoll) [inline Gemfile]: `elsif RUBY_VERSION < '2.2'`

**Review by mmoll — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 2 above)
Comment (ehelms) [reply Gemfile]: Oops, makes sense!

**Comment by ehelms** [comment]
Updated

**Review by mmoll — APPROVED** [review]

**Comment by mmoll** [comment]
Works as expected for mw now, I guess this should get cherry-picked all the way down to 1.20?

**Comment by ehelms** [comment]
I wasn't sure the implication for release branches. Right now this would fix a failure on CI.

**Comment by mmoll** [comment]
Let's unblock that run now :+1: 
If the problem is also seen in older branches, anybody should feel free to cherry-pick as needed.

**Comment by mmoll** [comment]
merged, thanks @ehelms!

**Pull request merged**